artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumProjectIntegration/Infrastructure/Models/RefreshResponse.cs

```diff
@@ -0,0 +1,10 @@
+using System.Text.Json.Serialization;
+
+namespace Artificial.Scrum.Master.ScrumProjectIntegration.Infrastructure.Models
+{
+    public class RefreshResponse
+    {
+        [JsonPropertyName("auth_token")] public string AccessToken { get; set; } = default!;
```

Comment:
jak nie chcesz warninga to ja polecam 
```suggestion
        [JsonPropertyName("auth_token")] public required string AccessToken { get; set; }